invokeai/frontend/web/src/features/gallery/components/ImageGalleryContent.tsx

```diff
@@ -81,6 +82,7 @@ const ImageGalleryContent = () => {
         </Tabs>
       </Flex>
 
+      <GallerySearch />
```

Comment:
Any ideas to keep this area of the UI as minimal as possible? Maybe next to the Images | Assets buttons, a separate search button that shows and hides the search box? Can iterate on this before release.

Reply:
Originally I was putting the selected number, sorting, and searching all at the top but I see you pulled the sort into the gallery menu instead. We could put an IconButton next to Images/Assets and have search field in a popover? And then the active search term as a hover state for button or something.

Reply:
Ya I moved it because selection number, sorting and search all in one tiny box is too dense. Plus we already have a popover for the settings.

The fewer tooltips/popovers, the better. When you have a search term set, you should see the box. Let's leave it like this for now, it's not too busy.

Reply:
It may look better once we have new boards UX too 👍 just pushed up other changes.